static/app/views/performance/newTraceDetails/traceDrawer/details/span/sections/generalInfo.tsx

```diff
@@ -113,10 +114,47 @@ export function GeneralInfo(props: GeneralnfoProps) {
     );
 
   let items: SectionCardKeyValueList = [
+    {
+      key: 'op',
+      subject: t('Op'),
+      value: props.node.value.op,
+      actionButton: (
+        <TraceDrawerComponents.KeyValueAction
+          rowKey={SpanIndexedField.SPAN_OP}
+          rowValue={props.node.value.op}
+        />
+      ),
+      actionButtonAlwaysVisible: true,
+    },
+    {
+      key: 'description',
+      subject: t('Description'),
+      value: props.node.value.description ? (
+        <DescriptionWrapper>
+          {getFormattedSpanDescription(props.node.value)}
+        </DescriptionWrapper>
```

Comment:
You may want some way to hover to see full description here

Reply:
@Zylphrex Yup, I've targetted the truncation only towards DB module descriptions and added the formatted overlay from the queries module